src/main/java/com/company/binarysearch/ContinuousTaskCompletion.java

```diff
@@ -0,0 +1,50 @@
+package com.company.binarysearch;
+
+public class ContinuousTaskCompletion {
+    public int getMinimumTime(int[] tasks, int k) {
+        int max = Integer.MIN_VALUE;
+        int sum = 0;
+        for (int arr : tasks) {
+            if (arr > max) {
+                max = arr;
+            }
+            sum += arr;
+        }
+        int left = max;
+        int right = sum;
+        int answer = 0;
+        while (left <= right) {
+            int mid = left + (right - left) / 2;
+            int total = tasks[0];
```

Comment:
Assign total = 0 and run for loop from 0 onwards. Your logic will not detect if arr[0] is less than mid

Reply:
changed